lib/Compiler.cpp

```diff
@@ -545,6 +545,16 @@ int RunPassPipeline(llvm::Module &M, llvm::raw_svector_ostream *binaryStream) {
     //   %3 = load float %2
     pm.addPass(llvm::createModuleToFunctionPassAdaptor(llvm::PromotePass()));
     pm.addPass(clspv::ClusterPodKernelArgumentsPass());
+
+    pm.addPass(clspv::InlineEntryPointsPass());
```

Comment:
It might be worth adding a pass of the function internalizer here too to avoid unnecessary work after inlining.

Reply:
Oh I see later on you've basically incorporated that pass into inline entry points. Not sure which way is better, so I'll leave to you.

Reply:
I have updated it to use function internalizer.